packages/mc-scripts/src/commands/start.ts

```diff
@@ -0,0 +1,129 @@
+import fs from 'fs';
+import webpack, { type Configuration } from 'webpack';
+import WebpackDevServer from 'webpack-dev-server';
+import chalk from 'chalk';
+import clearConsole from 'react-dev-utils/clearConsole';
+import checkRequiredFiles from 'react-dev-utils/checkRequiredFiles';
+import {
+  choosePort,
+  createCompiler,
+  prepareUrls,
+  type CreateCompilerOptions,
+} from 'react-dev-utils/WebpackDevServerUtils';
+import openBrowser from 'react-dev-utils/openBrowser';
+import paths from '../config/paths';
+import createDevServerConfig from '../config/webpack-dev-server.config';
+import createWebpackConfigForDevelopment from '../config/create-webpack-config-for-development';
+
+async function run() {
```

Comment:
Nothing much changed here, besides wrapping into `run` function, using async/await and cleaning up a bit.